source/features/linkify-issues-in-titles.tsx

```diff
@@ -1,17 +1,26 @@
 import select from 'select-dom';
+import zipTextNodes from 'zip-text-nodes';
 import features from '../libs/features';
 import observeEl from '../libs/simplified-element-observer';
+import parseBackticks from '../libs/parse-backticks';
 import {linkifyIssuesInDom} from './linkify-urls-in-code';
 
 function init(): void {
 	observeEl(
 		select('#partial-discussion-header')!.parentElement!,
 		() => {
-			const title = select('.js-issue-title:not(.refined-linkified-title)');
-			if (title) {
+			const title = select('.js-issue-title')!;
+
+			if (!title.classList.contains('refined-linkified-title')) {
```

Comment:
This change needs to be undone; the class ensures that what follows it is not repeated over and over after ajax updates. _"What follows it"_ now includes parsing the backticks.